assets/js/script.js

```diff
@@ -0,0 +1,164 @@
+const myAPIKey = "f23ee9deb4e1a7450f3157c44ed020e1";
+const currentWeather = $("#current-weather");
+const searchContainer = $("#search-history");
+const forecastContainer = $("#forecast-container");
+
+const constructSearchHistory = function (array) {
+  array.forEach((element) => {
+    historyButton = `<li><button id="search-btn">${element}</button></li>`;
+    const searchHist = $("#search-history");
+    searchHist.append(historyButton);
+  });
+};
+
+const renderSearchHistory = function () {
+  searchContainer.empty();
+  const array = getFromLocalStorage();
+  constructSearchHistory(array);
+};
+
+const constructTodaysWeather = function (data) {
+  const date = moment.unix(data.dt).format("DD-MM-YYYY");
+
+  return `<div>
+    <h2>${data.name} : ${date}
+        <img src="http://openweathermap.org/img/wn/${data.weather[0].icon}.png">
+    </h2>
+    <ul class="list-group-flush">
+        <li class="custom">Temperature: ${data.main.temp} <sup>O</sup>C</li>
+        <li class="custom">Humidity: ${data.main.humidity} %</li>
+        <li class="custom">Wind Speed: ${data.wind.speed} m/s</li>
+        <li class="custom" >UV Index: <span id="uvi"></span></li>
+      </ul>
+  </div>`;
+};
+
+const renderTodaysWeather = function (data) {
+  currentWeather.empty();
+  const topBlock = constructTodaysWeather(data);
+  currentWeather.append(topBlock);
+};
+
+const constructForecast = function (dailyArray) {
+  dailyArray.forEach((element) => {
+    const date = moment.unix(element.dt).format("DD-MM-YYYY");
+    forecastBlock = `<div class="card col-2 padstyle" style="width: 18rem;">
+    <div class="card-body">
+      <h5 class="card-title">${date}<img src="http://openweathermap.org/img/wn/${element.weather[0].icon}.png" alt="..."></h5>
+    </div>
+    <ul class="list-group list-group-flush">
+      <li class="list-group-item">Temperature: ${element.temp.day} <sup>O</sup>C</li>
+      <li class="list-group-item">Humidity: ${element.humidity} %</li>
+      <li class="list-group-item">Wind speed: ${element.wind_speed} m/s</li>
+    </ul>
+</div>`;
+    forecastContainer.append(forecastBlock);
+  });
+};
+
+const renderForecast = function (dailyArray) {
+  forecastContainer.empty();
+  constructForecast(dailyArray);
+};
+
+const getFromLocalStorage = function () {
+  const searchHistory = JSON.parse(localStorage.getItem("history")) || [];
+  return searchHistory;
+};
+
+const handleResponse = function (response) {
+  return response.json();
+};
+
+const showError = function () {
+  const error = document.createElement("p");
```

Comment:
Could we do this with jQuery instead to keep the code consistent?

Reply:
Yes I can